src/cpp/rtps/builtin/discovery/endpoint/EDPServer2.hpp

```diff
@@ -113,6 +112,9 @@ class EDPServer2 : public aux::EDPSimple
      */
     virtual bool createSEDPEndpoints() override;
 
+    //! TRANSIENT or TRANSIENT_LOCAL durability;
+    fastrtps::rtps::DurabilityKind_t _durability;
```

Comment:
And change it wherever is called.
```suggestion
    fastrtps::rtps::DurabilityKind_t durability_;
```